gatsby-node.js

```diff
@@ -5,21 +5,20 @@
  */
 
 // You can delete this file if you're not using it
-const path = require("path");
-const { createFilePath } = require("gatsby-source-filesystem");
+const path = require('path');
+const { createFilePath } = require('gatsby-source-filesystem');
 
 // convert windows to linux path
 const postDirectory = path.join(__dirname, 'posts').replace(/\\/g, '/');
 
-
-exports.createPages = async ({ actions: {createPage}, graphql }) => {
+exports.createPages = async ({ actions: { createPage }, graphql }) => {
   const postTemplate = path.resolve('src/components/post/postTemplate.js');
 
   const result = await graphql(`
     {
       allMarkdownRemark(
         sort: { order: DESC, fields: [frontmatter___date] }
-        filter: { fields: {slug: {regex: "\/posts\/[fall|winter|spring]/"}}}
+        filter: { fields: {slug: {regex: "/posts/[fall|winter|spring]/"}}}
```

Comment:
Unrelated to this PR but… what flavor is this regex? In JavaScript regex what you're looking for is probably `/\/posts\/(?:fall|winter|spring)\//`. Also note, GraphQL requires multiple escapes for backslashes according its documentation, so it should probably look like `regex: "/\\\\/posts\\\\/(?:fall|winter|spring)\\\\//"`

Also, shouldn't we group posts by year also?